src/main/kotlin/com/cosmotech/api/metrics/MonitorServiceAspect.kt

```diff
@@ -32,7 +32,9 @@ class MonitorServiceAspect(private var meterRegistry: MeterRegistry) {
           "connectorId")
 
   @Pointcut(
-      "within(@org.springframework.stereotype.Service *)" + " && within(com.cosmotech..*Impl)")
+      "within(@org.springframework.stereotype.Service *) && within(com.cosmotech..*Impl)" +
+          " && !execution(* com.cosmotech.scenariorun.azure.ScenarioRunServiceImpl.onScenarioDeleted(..))" +
+          " && !execution(* com.cosmotech.workspace.api.WorkspaceApiService.findWorkspaceById(..))")
```

Comment:
It is not the solution hard code a method name...

Reply:
I agree, it's not a good solution
But now we've no choice -> ticket for the next spring